src/covid_shared/cli_tools.py

```diff
@@ -143,6 +143,16 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self['start_time'] = datetime.datetime.now().strftime("%Y_%m_%d_%H_%M_%S")
 
+    def update(self, metadata_update: Mapping):
+        """Dictionary style update of metadata. If a value to store is the path to a metadata file, store that file's
+         contents instead."""
+        for key, value in metadata_update.items():
+            if type(value) == Path and value.name == paths.METADATA_FILE_NAME:
```

Comment:
NIT: `isinstance(value, Path)`.

But, the real thing is: don't alter the behavior of an exposed api method without warning.  Either:
1. Write some tests to show that this method behaves like the super method in all situations or 
2. make it a separate `update_from_path` method.